### PR TITLE
Add a setup.cfg

### PR DESCRIPTION
**Pull request by glasserc**
This should make it more likely that wheels are produced for this
package.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[zest.releaser]
+create-wheel = yes
```
Comment (peterbe) [inline setup.cfg]: I've never used zest.releaser. @leplatrem Can you check?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+[bdist_wheel]
+universal = 1
```
Comment (peterbe) [inline setup.cfg]: No new blank newline?

Comment (peterbe) [reply setup.cfg]: [Seems I lack a newline too](https://github.com/peterbe/hashin/blob/90f2804dc624a52cd9283bd62fead99b837a3c1f/setup.cfg#L1-L2) :)

Comment (glasserc) [reply setup.cfg]: I didn't understand what you mean by "new blank newline". Did you expect a blank line after the `[bdist_wheel]` header?

Comment (peterbe) [reply setup.cfg]: No, I meant a lone blank line as the last line. But now I see that there already is one. Ignore me.

File: setup.cfg (hunk 1 above)
Comment (leplatrem) [reply setup.cfg]: :+1:  eg. https://github.com/Kinto/kinto/blob/626201258a9143d76c95dc155bd8b4160ee3c684/setup.cfg#L4-L5

**Pull request merged**